keyboards/obosob/arch_36/keymaps/operezol/rules.mk

```
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = no       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no      # Enable keyboard RGB underglow
+OLED_ENABLE = no
+AUDIO_ENABLE = no           # Audio output
+SPLIT_KEYBOARD = yes       # Split common
+TAP_DANCE_ENABLE = yes
+
+LAYOUTS = split_3x5_3
```

You don't need, and really shouldn't have all of these listed.  Only what is different from default.   Eg, all you need is: 
```suggestion
EXTRAKEY_ENABLE = no       # Audio control and System control
OLED_ENABLE = no
TAP_DANCE_ENABLE = yes
```